android/assets/jsons/Civ V - Vanilla/Nations.json

```diff
@@ -1121,6 +1121,42 @@
 		"innerColor": [224,224,224],
 		"cities": ["Mogadishu"]
 	},
+	{
+		"name": "Bratislava",
+		"adjective": ["Bratislava"],
+		"cityStateType": "Religious",
```

Comment:
Preferably you make it so that religious city-states cannot spawn in games that have religion disabled

Reply:
the problem is that the game does not know if the game has religion disabled when generating the map

Reply:
It does, this is all five in startNewGame

Reply:
Now that religion is in beta anyways, its fine to remove all parts of `GameInfo.hasReligionEnabled` depending on the ruleset, everything should still work after that.

Reply:
So you've solved it by excluding the names of these city-states? That doesn't work because of mods: if you create a mod where any of these is a cultural city state, it will never show up. Similarly, if you create another religious city-state, it will show up in games without religion. why not do `.filter { it.cityStateType == Religious }` instead?

Reply:
@xlenstra got a better implementation